Airport Ticket Booking/Airport Ticket Booking/Commands/ManagerCommands/ValidationDetailsCommand.cs

```diff
@@ -0,0 +1,65 @@
+﻿using AirportTicketBooking.DBHandler;
+using AirportTicketBooking.Models;
+using AirportTicketBooking.Validation;
+using System.ComponentModel.DataAnnotations;
+using System.Reflection;
+using System.Text;
+
+namespace AirportTicketBooking.Commands.ManagerCommands
+{
+    internal class ValidationDetailsCommand : ICommandManager
+    {
+        public List<object> Execute(string[] parameters, FlightDataHandler flightDataHandler, BookingDataHandler bookingDataHandler)
+        {
+            string validationDetails = DesplayValidationFlightDetails(new Flight());
+            return new() { validationDetails };
+        }
+
+        private string DesplayValidationFlightDetails(Object obj)
```

Comment:
typo `Display', typos usually is a bad things since Devs usually use the method/class name to navigate and if there was a typo in the method/class name it would be harder on them to do so.

Also I would say we should name it `DisplayFlightValidationDetails/Rules'